packages/yarnpkg-cli/sources/boot-cli-dev.js

```diff
@@ -15,13 +15,30 @@ global.YARN_VERSION = require(`@yarnpkg/cli/package.json`).version;
 const PLUGIN_CONFIG_MODULE = `./tools/getPluginConfiguration.ts`;
 require.cache[require.resolve(PLUGIN_CONFIG_MODULE)] = {exports: {getPluginConfiguration}};
 
+const minimatch = require('minimatch');
```

Comment:
Can we use `micromatch`? Since we already use it in the core 🙂

Reply:
Done